util-boilerplate-creation/src/test/java/com/hpe/caf/util/boilerplate/creation/BoilerplateCreatorIT.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.util.boilerplate.creation;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.hpe.caf.boilerplate.webcaller.ApiClient;
+import com.hpe.caf.boilerplate.webcaller.ApiException;
+import com.hpe.caf.boilerplate.webcaller.api.BoilerplateApi;
+import com.hpe.caf.boilerplate.webcaller.model.BoilerplateExpression;
+import com.hpe.caf.boilerplate.webcaller.model.Tag;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Integration tests for boilerplate creation
+ */
+public class BoilerplateCreatorIT {
+    private static final String cafBoilerplateApiUrl;
```

Comment:
It wasn't, but I've updated the method that retrieves the ApiClient to use this property rather than retrieving the value from the environment each time it is called.